pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -3971,6 +3971,18 @@ final: prev:
     meta.homepage = "https://github.com/NTBBloodbath/galaxyline.nvim/";
   };
 
+  gen-nvim = buildVimPlugin {
+    pname = "gen.nvim";
+    version = "2024-05-03";
+    src = fetchFromGitHub {
```

Comment:
It is probably a good idea to create a patch for curl binary [here](https://github.com/David-Kunz/gen.nvim/blob/c9a73d8c0d462333da6d2191806ff98f2884d706/lua/gen/init.lua#L46) and [here](https://github.com/David-Kunz/gen.nvim/blob/c9a73d8c0d462333da6d2191806ff98f2884d706/lua/gen/init.lua#L55). Here is an example how to do that: https://github.com/NixOS/nixpkgs/blob/a4638fa3cbec7bdd5a79b212a75f1dc00957e011/pkgs/applications/editors/vim/plugins/overrides.nix#L1311-L1317